recipe/meta.yaml

```
@@ -15,83 +17,184 @@ source:
+        - eigen
+        - freetype
+    test:
+      commands:
+        - test -f $PREFIX/lib/libRDKitGraphMol.so              # [linux]
+        - test -f $PREFIX/lib/libRDKitGraphMol.dylib           # [osx]
+        - if not exist %LIBRARY_BIN%\RDKitGraphMol.dll exit 1  # [win]
+    about:
+      summary: RDKit C++ library
+      license: BSD-3-Clause
+      license_file: license.txt
+
+  - name: librdkit-dev
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - {{ stdlib("c") }}
+        - cmake
+      run:
+        - {{ pin_subpackage('librdkit', exact=True) }}
+    test:
+      commands:
+        - test -f $PREFIX/include/rdkit/GraphMol/GraphMol.h               # [unix]
+        - if not exist %LIBRARY_INC%\\rdkit\\GraphMol\\GraphMol.h exit 1  # [win]
+    about:
+      summary: RDKit headers and library used in librdkit
+      license: BSD-3-Clause
+      license_file: license.txt
+
+  - name: rdkit
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - {{ stdlib('c') }}
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+        - numpy                               # [build_platform != target_platform]
+        - python                              # [build_platform != target_platform]
+        - cmake
+        - jom   # [win]
+        - make  # [unix]
+        - pkg-config
+      host:
+        - cairo
+        - eigen
+        - freetype
+        - libboost-devel
+        - libboost-python-devel
+        - {{ pin_subpackage('librdkit', exact=True) }}
+        - matplotlib-base
+        - numpy
+        - pandas
+        - pillow
+        - pip
+        - postgresql
+        - pybind11-stubgen
+        - pycairo
+        - python
+        - reportlab
+        - setuptools
+        - setuptools_scm >=8
+        - sqlalchemy
+      run:
+        - {{ pin_subpackage('librdkit', exact=True) }}
+        - matplotlib-base
+        - numpy
+        - pandas
+        - pillow
+        - pycairo
+        - python
+        - reportlab
+        - sqlalchemy
+    test:
+      commands:
+        - python -c "import rdkit; assert rdkit.__version__ == '{{ version }}'"
+      imports:
         - rdkit
+        - rdkit.Avalon
+        - rdkit.Chem
+        - rdkit.Chem.AllChem
+        - rdkit.Chem.rdFreeSASA
+        - rdkit.DataManip
+        - rdkit.Dbase
+        - rdkit.DistanceGeometry
+        - rdkit.ForceField
+        - rdkit.Geometry
+        - rdkit.ML
+        - rdkit.Numerics
+        - rdkit.SimDivFilters
+        - rdkit.VLib
+        - rdkit.VLib.NodeLib
+        - rdkit.Chem.PandasTools
+    about:
+      summary: RDKit Python bindings
+      license: BSD-3-Clause
+      license_file: license.txt
+
+  - name: rdkit-postgresql
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - {{ stdlib("c") }}
```

```suggestion
        - {{ compiler('cxx') }}
        - {{ stdlib("c") }}
```

And this for the other `Needed DSO lib/libstdc++.so.6 found in ['conda-forge/linux-64::libstdcxx-ng==14.1.0=hc0a3c3a_0']` warning?

---

Maybe this is not needed after the libboost change in host, let's see 🤔

---

Turns out we will need it.